common/lib/dependabot/pull_request_creator/message_builder.rb

```diff
@@ -442,7 +442,6 @@ def maintainer_changes_cascade(dep)
       end
 
       def build_details_tag(summary:, body:)
-        # Azure DevOps does not support <details> tag (https://developercommunity.visualstudio.com/content/problem/608769/add-support-for-in-markdown.html)
         # CodeCommit does not support the <details> tag (no url available)
         if source_provider_supports_html?
```

Comment:
This has been moved to a different file: https://github.com/dependabot/dependabot-core/blob/main/common/lib/dependabot/pull_request_creator/message_builder/metadata_presenter.rb#L147-L157 so there are some merge conflicts that need to be fixed

Reply:
Fixed conflicts.